content/docs/faq.md

```diff
@@ -145,39 +145,39 @@ Anyway, back to the point. You can contribute in many ways, and some of it is re
 
   After that, you need to create a pull request. You can do so by forking [the repository](https://github.com/ScratchAddons/ScratchAddons/), do your necessary changes, and create a pull request. If it is deemed feasible, we will merge it.
 
-  We're also open for contributions of other aspects than the extension. You can view our repositories on [our GitHub organization page](https://github.com/ScratchAddons) and help us build them.
+  We're also open for contributions of other aspects than the extension. You can view our repositories on our [GitHub organization page](https://github.com/ScratchAddons) and help us build them.
 
 - **Suggest an idea**  
   
   Have something that you think would be a good addition to Scratch Addons? [Tell us!](#i-think-you-missed-a-feature-what-can-i-do)
 
 - **Report a bug**
   
-  Found a bug in one of our addon, the settings page, or anything else on our extension? [Send us a bug report](#what-can-i-do-if-i-find-a-problem).
+  Found a bug in one of our addon, the settings page, or anything else on our extension? Send us a [bug report](#what-can-i-do-if-i-find-a-problem).
```

Comment:
The full sentence describes the target page better.

Reply:
yeah, true